torch/functional.py

```diff
@@ -940,7 +940,10 @@ def tensordot(a, b, dims=2):
             raise RuntimeError(f"tensordot expects dims >= 0, but got dims={dims}")
         dims_a = list(range(-dims, 0))
         dims_b = list(range(dims))
-    return _VF.tensordot(a, b, dims_a, dims_b)  # type: ignore
+    if out is None:
+        return _VF.tensordot(a, b, dims_a, dims_b)  # type: ignore
+    else:
+        return _VF.tensordot(a, b, dims_a, dims_b, out=out) # type: ignore
```

Comment:
what does `type: ignore` mean?

Reply:
Thanks, it looks like it is for mypy static type checking. Looks we don't use it anymore? I removed it.